editoast/src/views/timetable/stdcm.rs

```diff
@@ -124,10 +125,13 @@ pub struct STDCMRequestPayload {
     #[schema(value_type = Option<String>, example = json!(["5%", "2min/100km"]))]
     margin: Option<MarginValue>,
     /// Total mass of the consist in kg
+    #[validate(range(min = 1.0))]
```

Comment:
Where does this `1.0` value comes from. I believe the ticket only mentions excluding `0.0`. You should be able to do that with the following.

```suggestion
    #[validate(range(exclusive_min = 0.0))]
```